### ISSUE TITLE
2 Player same phone?

### ISSUE THREAD

**Issue by pizza-beep** [lufinkey/iSSB#29]
There is a option to change cpu to human but you can't play that human
![Screenshot_20220302-005308](https://user-images.githubusercontent.com/67175855/156310631-32f51df3-a280-4e1a-b7cf-cfab0f99165c.png)

**Comment by lufinkey** [comment]
It's left over from the PC version, where you can have multiple players on one keyboard

**Comment by pizza-beep** [comment]
Oh
:0 Pc version?

**Comment by lufinkey** [comment]
Yeah when I was developing it I mainly was compiling for desktop because it
was easier to test

**Comment by pizza-beep** [comment]
Ok, thanks.
